megatron/model/__init__.py

```diff
@@ -13,7 +13,7 @@
         from .fused_rmsnorm import RMSNorm
     else:
         from .rmsnorm import RMSNorm
-    from torch.nn import LayerNorm
+    from .layernorm import LayerNorm
```

Comment:
Can you please share the failure that is being fixed? I have two concerns about this change: 
 1. It is quite subtle since it does not show the connection to sequence-parallelism
 2. It is unclear to me that new `LayerNorm` is equivalent to `torch.nn.LayerNorm` for non sequence-parallel case. 

So, I would like to further understand the problem and explore alternative solutions. Thanks!